src/app/conf/2025/components/testimonials/index.tsx

```diff
@@ -115,8 +115,7 @@ export function TestimonialAuthor({
           width={128}
           height={128}
           className="size-16 saturate-[.1] dark:opacity-90 xl:size-32"
-          // @ts-expect-error React doesn't know it exists yet, but @types/react do
-          fetchpriority="low"
+          fetchPriority="low"
```

Comment:
Did you bump the React version? so... if it's not lowercase and React doesn't know about it, will it still get passed?